pages/pipelines/migrate_from_jenkins.md

```diff
@@ -0,0 +1,204 @@
+# Migrate from Jenkins
+
+If you are familiar with [Jenkins](https://www.jenkins.io) and want to migrate to Buildkite, this guide is for you. Buildkite is a modern and flexible continuous integration and delivery (CI/CD) platform that provides a powerful and scalable build infrastructure for your applications.
+
+While Jenkins and Buildkite have similar goals as CI/CD platforms, their approach differs. Buildkite uses a hybrid model consisting of:
+
+- A software-as-a-service (SaaS) platform for visualization and management of CI pipelines.
```

Comment:
Should this be CI/CD pipelines? Or simply, pipelines?

Reply:
Hmmm good question. I think just "pipelines" is nicer, but we haven't talked about it like that yet, so will go with the longer one here.

```suggestion
- A software-as-a-service (SaaS) platform for visualization and management of CI/CD pipelines.
```